.github/workflows/benchmark-trigger.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 concurrency:
   # Group by flow, PR number, and a trigger condition (so other labels don't cancel an in-progress job)
-  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.action != 'labeled' || github.event.label.name == 'action:run-benchmark' }}
```

Comment:
What is the meaning of this change?

Reply:
grouping by a PR event (opened, sync...), or by the specific label that was added, so the benchmark and micro-benchmark flows can run in parallel.
But actually I think it will be better to have concurrency per job